### PR TITLE
move animated gif checker to utilities

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

Move `isDataAnimatedGIF` to utilities 

## dependency
- [x] https://github.com/wireapp/wire-ios-utilities/pull/124

### CODE REVIEW

File: Cartfile.resolved [hunk 1]
```diff
@@ -8,5 +8,5 @@ github "wireapp/wire-ios-link-preview" "31.0.4"
 github "wireapp/wire-ios-protos" "28.0.1"
 github "wireapp/wire-ios-system" "34.1.1"
 github "wireapp/wire-ios-testing" "25.1.1"
-github "wireapp/wire-ios-transport" "77.1.2"
-github "wireapp/wire-ios-utilities" "41.1.0"
+github "wireapp/wire-ios-transport" "fix/rm-uttype"
```
Comment (billypchan) [inline Cartfile.resolved:11]:
```suggestion
github "wireapp/wire-ios-transport" "78.0.0"
```

File: Cartfile.resolved [hunk 2]
```diff
@@ -8,5 +8,5 @@ github "wireapp/wire-ios-link-preview" "31.0.4"
 github "wireapp/wire-ios-protos" "28.0.1"
 github "wireapp/wire-ios-system" "34.1.1"
 github "wireapp/wire-ios-testing" "25.1.1"
-github "wireapp/wire-ios-transport" "77.1.2"
-github "wireapp/wire-ios-utilities" "41.1.0"
+github "wireapp/wire-ios-transport" "78.0.0"
+github "wireapp/wire-ios-utilities" "fix/rm-uttype"
```
Comment (billypchan) [inline Cartfile.resolved:12]:
```suggestion
github "wireapp/wire-ios-utilities" "fix/mv-animated-gif-checker"
```

File: Source/Model/Message/ZMMessage.swift [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+
+import Foundation
+
+extension ZMMessage {
```
Comment (KaterinaWire) [inline Source/Model/Message/ZMMessage.swift:22]: I guess we don't need this file.

Comment (billypchan) [reply Source/Model/Message/ZMMessage.swift]: cleared

**Review by KaterinaWire — APPROVED** [review]

**Review by johnxnguyen — APPROVED** [review]

**Comment by billypchan** [comment]
@zenkins release minor

**Pull request merged**